tests/_bootstrap.php

```diff
@@ -0,0 +1,6 @@
+<?php
+defined('APPLICATION_ENV') || define('APPLICATION_ENV', 'devtest');
+defined('APPLICATION_STORE') || define('APPLICATION_STORE', 'DE');
+defined('APPLICATION_ROOT_DIR') || define('APPLICATION_ROOT_DIR',  '/data/shop/development/current');
+defined('APPLICATION') || define('APPLICATION', 'ZED');
+defined('APPLICATION_VENDOR_DIR') || define('APPLICATION_VENDOR_DIR',  realpath(__DIR__));
```

Comment:
I don`t really get the point. I have checked 
https://github.com/spryker-eco/payone/tree/master/tests
https://github.com/spryker-eco/billpay/tree/bugfix/eco-ECO-635/tests
They have such structure. Can You please provide an example?